### PR TITLE
Fixes: minor fixes on products and clients

### PR DESCRIPTION
**Pull request by daudujohn**
# What does this PR do...

- Filters products based on exclusivity
- Defaults backgroundColor options to true
- Populate image field for an individual client request

### CODE REVIEW

File: gpb2.0/types/generated/contentTypes.d.ts
Comment (RICHYDOS) [inline gpb2.0/types/generated/contentTypes.d.ts:1]: What is going on in this file???

**Review by RICHYDOS — CHANGES_REQUESTED** [review]

File: gpb2.0/types/generated/contentTypes.d.ts
Comment (daudujohn) [reply gpb2.0/types/generated/contentTypes.d.ts:1]: My bad, I hadn't installed EmailDesigner package, so i thought the change came from adding a filter on exclusivity.
I will fix it now.

Comment (RICHYDOS) [reply gpb2.0/types/generated/contentTypes.d.ts:1]: okay then

**Review by RICHYDOS — APPROVED** [review]
Niceee

**Pull request merged**